src/presets/firebase/utils.ts

```diff
@@ -47,9 +47,9 @@ export async function updatePackageJSON(nitro: Nitro) {
         .sort(([a], [b]) => a.localeCompare(b))
     ),
     engines: {
+      // https://cloud.google.com/functions/docs/runtime-support
       // https://cloud.google.com/functions/docs/concepts/nodejs-runtime
-      // const supportedNodeVersions = new Set(["20", "18", "16"]);
```

Comment:
I think this was from the old preset. I've removed it since it's no longer used.